<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>T4auto_calib_wizard</class>
 <widget class="QWizard" name="T4auto_calib_wizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>542</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>487</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Wizard-tytul</string>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ClassicStyle</enum>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <property name="title">
    <string>Constructing a skeleton for spectra names</string>
   </property>
   <property name="subTitle">
    <string>Here you can tell the system which spectra have to be analysed (fitted) to create their calibration.</string>
   </property>
   <attribute name="pageId">
    <string notr="true">1</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="20,0">
          <item>
           <widget class="QLabel" name="textLabel1_2">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Type below the spectrum name. In the name you can &lt;br/&gt;use symbols %1, %2, %3 to mark places, which will be changing. &lt;br/&gt;&lt;br/&gt;For example: &lt;span style=&quot; font-size:large; font-weight:600; color:#0000aa;&quot;&gt;detector_&lt;/span&gt;&lt;span style=&quot; font-size:large; font-weight:600; color:#0000ff;&quot;&gt;%1&lt;/span&gt;&lt;span style=&quot; font-size:large; font-weight:600; color:#0000aa;&quot;&gt;_&lt;/span&gt;&lt;span style=&quot; font-size:large; font-weight:600; color:#ff0000;&quot;&gt;%2&lt;/span&gt;&lt;span style=&quot; font-size:large; font-weight:600; color:#0000aa;&quot;&gt;_&lt;/span&gt;&lt;span style=&quot; font-size:large; font-weight:600; color:#00aa00;&quot;&gt;%3&lt;/span&gt;&lt;span style=&quot; font-size:large; font-weight:600; color:#0000aa;&quot;&gt;_energy_raw.spc&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="_2">
          <item>
           <widget class="QLabel" name="textLabel_skeleton">
            <property name="text">
             <string>Skeleton  for a spectrum name</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_skeleton"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="20,1">
          <item>
           <widget class="QLabel" name="textLabel3">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Prepare lists of symbols. The items on the list should be separated by spaces. &lt;br/&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;For example &lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:10pt; color:#0000ff;&quot;&gt;%1&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;: A B C D&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:10pt; color:#ff0000;&quot;&gt;%2&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;: x y&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:10pt; color:#00aa00;&quot;&gt;%3&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;: 00 01 02 03 04 05 06 07&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="_3">
          <item>
           <widget class="QLabel" name="textLabel2">
            <property name="text">
             <string>&lt;font color=&quot;#0000ff&quot;&gt;&lt;b&gt;%1&lt;/b&gt;:&lt;/font&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_one"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="_4">
          <item>
           <widget class="QLabel" name="textLabel2_2">
            <property name="text">
             <string>&lt;b&gt;&lt;font color=&quot;#ff0000&quot;&gt;%2&lt;/font&gt;:&lt;/b&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_two"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="_5">
          <item>
           <widget class="QLabel" name="textLabel2_3">
            <property name="text">
             <string>&lt;b&gt;&lt;font color=&quot;#00aa00&quot;&gt;%3&lt;/font&gt;:&lt;/b&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_three"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="spacer5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>481</width>
            <height>21</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Skeleton</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="pushButton_restore_skeleton">
                <property name="text">
                 <string>Restore</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_save_skeleton">
                <property name="text">
                 <string>Save as...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_skel">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="textLabel4">
          <property name="text">
           <string>List of spectra names created by your skeleton</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_show_spectra_names">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>Produce the list 
of spectra names</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit_spectra_names"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <property name="title">
    <string>You are going to make an automatic calibration of the all germanium detectors</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="_10">
      <item>
       <layout class="QHBoxLayout" name="_11">
        <item>
         <widget class="QLabel" name="textLabel1_5">
          <property name="text">
           <string>Name of file describing peaks in your calibration source       *.sou</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_source_description_file"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="spacer4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>281</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="_9">
      <item>
       <widget class="QLabel" name="textLabel1">
        <property name="text">
         <string>Start searching for peaks (in the RAW spectra) from channel:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox1_thrsh">
        <property name="maximum">
         <number>8192</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="textLabel1_4">
      <property name="text">
       <string>&lt;b&gt;The calibration routine takes the current state of spectra from the disk.&lt;br&gt;
So - if your analysis is running - be sure that all the spectra have been recently saved&lt;/b&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="textLabel1_4_2">
      <property name="text">
       <string>&lt;p align=&quot;right&quot;&gt;The spectra are always saved when you &lt;b&gt;&lt;font color=&quot;#aa0000&quot;&gt;stop/pause&lt;/font&gt;&lt;/b&gt; the analysis&lt;/p&gt;
&lt;p align=&quot;right&quot;&gt;or when you press the &lt;b&gt;&lt;font color=&quot;#aa0000&quot;&gt;update&lt;/font&gt;&lt;/b&gt; button&lt;/p&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="spacer38">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>109</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="push_run_gf4">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>78</height>
       </size>
      </property>
      <property name="text">
       <string>run gf5 program</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="textLabel_note_about_silence_gf4">
      <property name="maximumSize">
       <size>
        <width>32767</width>
        <height>83</height>
       </size>
      </property>
      <property name="text">
       <string>Note: when you press this button - nothing seems to happen - but looking
at the window where from you started &quot;cracow&quot; - you will see all output form the Carl's gf5  program.
Soon you will notice a graphic window appearing and blinking on the screen.</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>push_run_gf4</zorder>
   <zorder>textLabel1_4</zorder>
   <zorder>textLabel1_4_2</zorder>
   <zorder>textLabel_note_about_silence_gf4</zorder>
  </widget>
  <widget class="QWizardPage" name="wizardPage">
   <property name="title">
    <string>Here you can watch the file with error messages</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPlainTextEdit" name="textEdit_error_messages">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(252, 255, 190);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage_2">
   <property name="title">
    <string>Result of the autocalibration - the new coefficients</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QPlainTextEdit" name="textEdit_results">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(232, 255, 219);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage_3">
   <property name="title">
    <string>Install</string>
   </property>
   <property name="subTitle">
    <string>Installing the coefficients - means that the new coeffincients will be introduced into the &lt;font color=&quot;#00aa00&quot;&gt;&lt;tt&gt;./calibration/rising_calibration.txt&lt;/tt&gt;&lt;/font&gt; file (replacing the former ones)</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="textLabel1_3">
        <property name="text">
         <string>&lt;font size=&quot;+2&quot;&gt;&lt;b&gt;If you want to install these new coefficients, &lt;br&gt;please press the button &quot;Install&quot;&lt;/b&gt;&lt;/font&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_13">
        <item>
         <layout class="QVBoxLayout" name="_14">
          <item>
           <widget class="QPushButton" name="push_install">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>52</height>
             </size>
            </property>
            <property name="text">
             <string>Install new coefficients</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_archive">
            <property name="text">
             <string>Yes, please make archive</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="textLabel_archive">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;to archive - means:&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;&lt;br/&gt;&lt;br/&gt;a) the previous calibration file into the subdirectory &amp;quot;./calibration/history&amp;quot; &lt;br/&gt;with the current file name appended with text&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New,courier'; font-size:9pt;&quot;&gt;rising_calibration_&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New,courier'; font-size:9pt; color:#ff0000;&quot;&gt;till&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New,courier'; font-size:9pt;&quot;&gt;_&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New,courier'; font-size:9pt; font-style:italic; color:#0000ff;&quot;&gt;current_date_time&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New,courier'; font-size:9pt;&quot;&gt;.txt&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;&lt;br/&gt;&lt;br/&gt;b) The new calibration will be stored as &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New,courier'; font-size:9pt;&quot;&gt;rising_calibration_&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New,courier'; font-size:9pt; color:#ff0000;&quot;&gt;since&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New,courier'; font-size:9pt;&quot;&gt;_&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New,courier'; font-size:9pt; font-style:italic; color:#0000ff;&quot;&gt;current_date_time&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New,courier'; font-size:9pt;&quot;&gt;.txt&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>121</width>
          <height>41</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_install_messages">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>12</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
